apps/balancer-tools/src/app/apr/(utils)/getFilteredApiUrl.ts

```diff
@@ -4,6 +4,13 @@ import { SearchParams } from "../round/[roundId]/page";
 export const INITIAL_MIN_TVL = 1000;
 export const INITIAL_LIMIT = 10;
 
+interface ExpectedSearchParams extends SearchParams {
+  minTVL?: string | undefined;
```

Comment:
```suggestion
  minTVL?: string | undefined;
```
is redundant. You can just use `prop?: string`: https://www.typescriptlang.org/docs/handbook/2/functions.html#optional-parameters